### PR TITLE
Update mp

### PR DESCRIPTION
**Pull request by qwe789qwec**
Please Check

### CODE REVIEW

**Comment by pedrobcst** [comment]
It seems that the newest versions of scipy have moved pinv2 used by peakutils somewhere else.

Can you fix the scipy version to 1.5.3 and see if the tests can pass?

**Comment by qwe789qwec** [comment]
let me fix it later.
Why does it pass on my computer but not online?
what's different?

File: Xerus/queriers/multiquery.py [hunk 1]
```diff
@@ -97,32 +97,33 @@ def multiquery(element_list: List[str], max_num_elem: int, name: str, resync:boo
...
+            max_num_elem=max_num_elem,
+            min_hull=0.1, 
+            folder_path=mp_path) # Moving hull to 0.1
     td.append(mp_path)
 
     ## COD query
+    cod_path = str(os.path.join(abs_path, f"{name}_{query_type}_COD")) + os.sep
     cod = CODQuery(elements=element_list,
                    max_num_elements=max_num_elem,
-                   combination=False,
```
Comment (pedrobcst) [inline Xerus/queriers/multiquery.py:121]: Why the combination was removed from CODQuery? Its by default True, and in this method we are doing it should always be False (messy code, I am sorry). Can you double check this?

File: tests/test_settings.py [hunk 2]
```diff
@@ -7,8 +7,10 @@
 import GSASIIscriptable as G2sc
 from Xerus.db.localdb import LocalDB
 from Xerus.settings.settings import MP_API_KEY
-from pymatgen.ext.matproj import MPRestError
-from pymatgen import MPRester
+# from pymatgen.ext.matproj import MPRestError
+# from pymatgen.ext.matproj import MPRester
```
Comment (pedrobcst) [inline tests/test_settings.py:11]: Remove the old packages from comment. Not needed.

File: Xerus/queriers/mp.py [hunk 3]
```diff
@@ -76,34 +75,21 @@ def querymp(inc_eles: List[str], exc_eles: List = [], max_num_elem:int = 3, api_
...
-        return final
+    properties = ['formula_pretty',  'material_id',  'structure',  'energy_above_hull',  'theoretical', 'fields_not_requested']
 
-    if len(data) == 0:
+    mpr = MPRester(api_key)     
+    datas = mpr.summary.search( chemsys=inc_eles,
+                                fields = properties, 
+                                theoretical = False, 
+                                energy_above_hull = ( 0, min_hull))
+    if "pytest" in sys.modules:
```
Comment (pedrobcst) [inline Xerus/queriers/mp.py:85]: Please explain why this is necessary

File: Xerus/queriers/multiquery.py (hunk 1 above)
Comment (qwe789qwec) [reply Xerus/queriers/multiquery.py:121]: I think I only change the mp part.
But let me check that later.

**Review by pedrobcst — CHANGES_REQUESTED** [review]
Please address the issues I raised, then it should be g2g.

Mostly check the COD querier again.
I have fixed for now scipy==1.5.3 since pinv2 moved somewhere else
Please explain those changes you made

**Review by pedrobcst — APPROVED** [review]

**Pull request merged**